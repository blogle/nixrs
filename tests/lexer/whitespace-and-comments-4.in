# line comment
  1 ### lex this please
13 
 42
# end of file
