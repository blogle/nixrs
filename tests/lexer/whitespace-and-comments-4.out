[1:1-1:15] (Comment) "# line comment"
[2:3-2:4] (Integer) "1"
[2:5-2:24] (Comment) "### lex this please"
[3:1-3:3] (Integer) "13"
[4:2-4:4] (Integer) "42"
[5:1-5:14] (Comment) "# end of file"
